podcasts/climate-mit-edu.yaml

```diff
@@ -1,5 +1,5 @@
 ---
 rss_url: https://feeds.simplecast.com/w5_4mil2
-offered_by: MIT Climate
+offered_by:
```

Comment:
If the offered by is none you should omit the field entirely instead of leaving it blank

Reply:
Thanks! Should be all set now.